python/cugraph/cugraph/tests/test_property_graph.py

```diff
@@ -1750,11 +1750,13 @@ def test_renumber_vertices_by_type(dataset1_PropertyGraph, prev_id_column):
         assert df_id_ranges.loc[key, "stop"] == stop
         df = pG.get_vertex_data(types=[key])
         assert len(df) == stop - start + 1
-        assert (df["_VERTEX_"] == list(range(start, stop + 1))).all()
+        assert (
```

Comment:
These pytest fixes were needed because `cudf` removed the comparison of host & device objects in : https://github.com/rapidsai/cudf/pull/12281/, which will return `False` if performed.